UACloudLibraryServer/QueryModel.cs

```diff
@@ -47,183 +48,102 @@ namespace Opc.Ua.Cloud.Library
     public class QueryModel
     {
         [UsePaging, UseFiltering, UseSorting]
-        public IQueryable<CloudLibNodeSetModel> GetNodeSets([Service(ServiceKind.Synchronized)] AppDbContext dbContext, string identifier = null, string nodeSetUrl = null, DateTime? publicationDate = null)
+        public IQueryable<CloudLibNodeSetModel> GetNodeSets([Service(ServiceKind.Synchronized)] CloudLibDataProvider dp, string identifier = null, string nodeSetUrl = null, DateTime? publicationDate = null)
```

Comment:
Please make sure Markus takes a look at this before merging.

Reply:
Yes, this looks good!